conf/application.conf

```diff
@@ -171,7 +171,7 @@ mongo-async-driver {
 
 urls {
   login = "http://localhost:9949/auth-login-stub/gg-sign-in"
-  loginContinue = "http://localhost:9000/customs-declare-exports-frontend"
+  loginContinue = "http://localhost:6791/customs-declare-exports-frontend/start"
```

Comment:
hmm, I think it should be: `http://localhost:6791/customs-declare-exports/start` without frontend. We're not using frontend in the name from few months

Reply:
Good point. I think it only worked for me locally because auto-submit overrode it...